### DISCUSSION TITLE
关于断连后自动重启bot的一些想法

### DISCUSSION THREAD

**Discussion by black-zero358** [nonebot/discussions#527] [通用]
![image](https://github.com/user-attachments/assets/811d435b-3965-4787-a233-1d77860e55ea)
在想可不可写个插件检测断连之后nonebot这边的消息，然后自动去重启Lagrange.OneBot

**Comment by black-zero358** [comment]
好像systemd 也能完成这样的需求

**Comment by yanyongyu** [comment]
你可以通过子进程启动lgr，然后检测进程状态，当然不是很推荐这么做
